extensions/sql/syntaxes/sql.tmLanguage.json

```diff
@@ -4,7 +4,7 @@
 		"If you want to provide a fix or improvement, please create a pull request against the original repository.",
 		"Once accepted there, we are happy to receive an update request."
 	],
-	"version": "https://github.com/Microsoft/vscode-mssql/commit/cfc8b65ed6daf8252b0cbfd5611aadbd49353bca",
+	"version": "https://github.com/Microsoft/vscode-mssql/commit/c8effddd6a9df117f3ed45b60b487163950a7ea5",
```

Comment:
is there a way to add the {{SQL CARBON EDIT}} comment in this file?  otherwise, I'm afraid it may be missed in next VS Code merge?  is the comment creates a syntax error in JSON file then perhaps we should add the comment into a text file in the extension with a note to update this field during merge, or something like that.

Reply:
Any sql syntax changes should be applied via an update script specified in https://github.com/Microsoft/sqlopsstudio/blob/master/extensions/sql/package.json , which also updates this version field. I hadn't changed this field in the previous commit because I wasn't aware of that update workflow, and ended up doing the PRs in the reverse order. Currently VSCode has all the syntax changes up to this point (although they are one commit ahead), so there's no SqlOps divergence here.

Reply:
https://github.com/Microsoft/vscode/blob/master/extensions/sql/syntaxes/sql.tmLanguage.json#L7